src/site/_includes/i18n-index.njk

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
let's not use i18n in the name of the partial, since it has nothing to do with intenationalisation of the site. We can use something like 'index-page' (it renders an index of items) and 'item-page' (renders single item).